compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeScene.skiko.kt

```diff
@@ -243,6 +244,22 @@ interface ComposeScene {
      */
     fun sendKeyEvent(keyEvent: KeyEvent): Boolean
 
+    /**
+     * Send rotary scroll event to the content.
+     * @param verticalScrollPixels The amount to scroll (in pixels) in response to a
+     * [RotaryScrollEvent] in a container that can scroll vertically.
+     * @param horizontalScrollPixels The amount to scroll (in pixels) in response to a
+     * [RotaryScrollEvent] in a container that can scroll horizontally.
+     * @param timeMillis The time in milliseconds at which this even occurred. The start (`0`) time
+     * is platform-dependent.
+     * @return true if the event was consumed by the content
+     */
+    fun sendRotaryScrollEvent(
+        verticalScrollPixels: Float,
+        horizontalScrollPixels: Float,
+        timeMillis: Long = currentTimeForEvent(),
+    ): Boolean
```

Comment:
`RotaryScrollEvent` is public, so no need to hide the type here like we had to do for `sendPointerEvent`

```suggestion
    fun sendRotaryScrollEvent(rotaryScrollEvent: RotaryScrollEvent): Boolean
```

Reply:
Sure, however, the constructor of RotaryScrollEvent is internal in skiko sourceset. 

https://github.com/JetBrains/compose-multiplatform-core/blob/a476deb771f9810965c934b0bb286c52576dcdd2/compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/input/rotary/RotaryScrollEvent.skiko.kt#L25

Is it alright to make this public?

On Android. this event is already forwarded by the system so the user does not need to instantiate this class. But for multiplatform this would need to be public.

Reply:
Hmm, indeed. Then the current shape is fine.